core/templates/components/skill-selector/skill-selector.component.html

```diff
@@ -0,0 +1,120 @@
+<div class="skill-selector">
+  <div class="skill-selector-parent skill-container oppia-skill-container protractor-test-skill-container">
+    <div class="list-group oppia-list-group">
+      <mat-card layout="row" class="list-view-item protractor-test-skill-select-header">
+        Search By Skill
+        <input type="text" class="form-control protractor-test-skill-name-input" placeholder="Enter a skill name" [(ngModel)]="skillFilterText" autofocus>
+      </mat-card>
+      <div class="skill-list-box">
+        <mat-radio-group [(ngModel)]="selectedSkill" (change)="setSelectedSkillId()">
+          <div *ngFor="let topic of currCategorizedSkills | keyvalue">
+            <!-- topic.key = topicName, topic.value = subtopics -->
+            <mat-card class="list-view-item" *ngIf="checkTopicIsNotEmpty(topic.key)">
+              <mat-card layout="row" class="list-view-item">
+                <h2><label class="oppia-topic-name-text">{{ topic.key }}</label></h2>
+              </mat-card>
+              <div *ngFor="let subtopic of topic.value | keyvalue" class="list-view-item ">
+                <!-- subtopic.key = subtopicName (string) / uncategorized, subtopic.value = skills (ShortSkillSummary[]) -->
+                <div *ngIf="!checkIfEmpty(subtopic.value)">
+                  <mat-card class="list-view-item">
+                    <label class="oppia-subtopic-name-text">{{ subtopic.key }}</label>
+                    <mat-radio-button class="oppia-skills-list-item protractor-test-skills-list-item" *ngFor="let skill of searchInSubtopicSkills(subtopic.value, skillFilterText)" [value]="skill.id" aria-label="{{ skill.description }}">
```

Comment:
change `aria-label` to `attr.aria-label`

Reply:
Done